packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -587,12 +617,16 @@ export class ProjectService {
         const adapter = await this.getAdapter(projectUuid);
         const { rows } = await adapter.runQuery(query);
 
-        const formattedRows = formatRows(
-            rows,
+        const itemMap = getItemMap(
             explore,
             metricQuery.additionalMetrics,
             metricQuery.tableCalculations,
         );
+        const formattedRows = await wrapSentryTransaction(
```

Comment:
we can remove the `wrapSentryTransaction` usage here (or not), I added it for testing